### ISSUE TITLE
[survey_accounts] Email survey not working as expected

### ISSUE THREAD

**Issue by sruthymathew123** [aces/Loris#4527]
[Tested on test.loris.ca](https://test.loris.ca/)

I am getting a blank page(500 error) after hitting "Email Survey" button
 **I didn't get any notification Email (but the survey got created in survey_accounts)**

**Comment by johnsaigle** [comment]
This may be related to #4515. It seems that email is not working on the test server.

**Comment by sruthymathew123** [comment]
@ridz @jesscall 

I retested this on my VM and I was to able to reproduce the error above. 
 I could identify the issue with the error log. The issue was actually with the instrument I tried to send as survey.  
Fixing the instrument (fixing the function compatibility issues and using `  use \LegacyInstrumentTrait;` within the instrument fixed the issue. I could successfully send the survey (I got email notification as well :) )
![screenshot](https://user-images.githubusercontent.com/23702452/57937317-aed17900-7893-11e9-8058-1581a1861bee.png)


@ridz Can I close this issue or you guys need any further testing on CCNA?
